tasks/main.yml

```diff
@@ -97,24 +97,13 @@
   file:
     dest: /etc/resolv.conf
     state: link
-    src: "{{ _systemd_network_resolv_conf_paths[systemd_network_dns_resolver] }}"
+    src: /run/systemd/resolve/resolv.conf
     force: True
     follow: False
 
-- name: Populate service facts
-  service_facts: {}
-
-- name: Disable unwanted resolvers
```

Comment:
Sure we do not want to disable resolveconf in case it is active?

Reply:
Right.